### ISSUE TITLE
Example context needs updating

### ISSUE THREAD

**Issue by dmitrizagidulin** [w3c/vc-data-model#622]
From @yancyribbens's [original issue at vc-test-suite](https://github.com/w3c/vc-test-suite/issues/19):

The context: https://www.w3.org/2018/credentials/examples/v1 is invalid JSON. If you paste the context into https://jsonlint.com/ the following parse error occurs:

```
Error: Parse error on line 24:
...			"schema:alumniOf",			"@type": rdf: H
-----------------------^
Expecting ':', got ','
```

Moving it here, since this is where the `examples/v1` context is hosted. (A valid examples context can be found at https://github.com/digitalbazaar/vc-js/blob/master/lib/contexts/vc-examples-v1.js , if that's helpful)

**Comment by dmitrizagidulin** [comment]
Fix pending at PR #623.

**Comment by msporny** [comment]
PR #623 is merged, we should close this issue.

**Comment by burnburn** [comment]
WG resolution on 28 May 2019 call:

RESOLUTION: Issue #622 is resolved via PR #623, which fixes a bug in the JSON-LD context. Issue #622 should be closed immediately.
